### PR TITLE
Dev1.2

### PR DESCRIPTION
**Pull request by Shilpa-Manjunath**

### CODE REVIEW

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/queue/impl/MosipActiveMqImpl.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
     /**
      * The reg proc logger.
      */
-    private static Logger regProcLogger = RegProcessorLogger.getLogger(MosipActiveMqImpl.class);
+    private static org.apache.logging.log4j.Logger regProcLogger = RegProcessorLogger.getLogger(MosipActiveMqImpl.class);
```
Comment (MonobikashDas) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/queue/impl/MosipActiveMqImpl.java:39]: why did you change kernel logger to apache logger?

Comment (Shilpa-Manjunath) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/queue/impl/MosipActiveMqImpl.java:39]: No due to local setup issue this has happened but I have reverted it
